packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -191,7 +175,14 @@ function NavigationMenuItemEdit( {
 					'is-selected': isSelected,
 				} ) }
 			>
-				{ content }
+				<RichText
+					ref={ plainTextRef }
```

Comment:
What was the original purpose of this? Is is still needed?

Reply:
Initially, we used a plain text and had a button to focus the plain text, so we needed a ref to focus the field. This reference is not needed anymore as the button was removed.

Reply:
It doesn't seem to be used